### ISSUE TITLE
i18n for "_canAddItem" function

### ISSUE THREAD

**Issue by froggiefab** [jshjohnson/Choices#69]
Hi,

Could you provide a way (in the configuration object maybe ?) to customize labels used in the "_canAddItem" function ?

Template string functionnality is used for two of them and I don't know how to best handle those cases to propose a PR...

Thanks

**Comment by jshjohnson** [comment]
To achieve this there would need to be a config option that accepts either a string or a function. The function would have a value argument that could then be used within the returned label. 

So something like: 
```js
const stringExample = new Choices(element, {
  addItemLabelText: "Press enter to add value",
});

const functionExample = new Choices(element, {
  addItemLabelText: (value) => {
    return `Press enter to add ${value}`
  },
});
```

**Comment by froggiefab** [comment]
Thanks, will submit a PR soon :)

**Comment by jshjohnson** [comment]
This has now been merged into [version 2.5.0](https://github.com/jshjohnson/Choices/releases/tag/v2.5.0).

Thanks
